sway/input/cursor.c

```diff
@@ -982,6 +983,18 @@ static void handle_cursor_button(struct wl_listener *listener, void *data) {
 	transaction_commit_dirty();
 }
 
+static uint32_t wl_axis_to_button(struct wlr_event_pointer_axis *event) {
+	switch (event->orientation) {
+	case WLR_AXIS_ORIENTATION_VERTICAL:
+		return event->delta < 0 ? KEY_MAX + 1 : KEY_MAX + 2;
```

Comment:
Can we have `#define`s for those special values?

Reply:
Done